packages/components/src/Content/Content.tsx

```diff
@@ -133,11 +133,12 @@ const TitleContainer = styled.div<{ titleColor?: keyof typeof colors }>`
   color: ${({ theme, titleColor }) => titleColor && theme.colors[titleColor]};
 `
 
-const Title = styled.div`
+const Title = styled.div<{ titleOverflow?: boolean }>`
   ${({ theme }) => theme.fonts.h2}
   color: ${({ theme }) => theme.colors.copy};
-  overflow: hidden;
-  text-overflow: ellipsis;
+  overflow: ${(props) => (props.titleOverflow ? 'auto' : 'hidden')};
+  overflow-y: ${(props) => (props.titleOverflow ? 'hidden' : '')};
+  text-overflow: ${(props) => (props.titleOverflow ? '' : 'ellipsis')};
```

Comment:
will this output invalid CSS in some cases? like `overflow-y: ;`

Reply:
yeah probably should not let invalid css to happen. will fix